docs/reference-guides/block-api/block-selectors.md

```diff
@@ -0,0 +1,116 @@
+# Selectors
+
+Block Selectors is the API that allows blocks to customize the CSS selector used
+when their styles are generated.
```

Comment:
```suggestion
Block Selectors is an API that allows blocks to customize the CSS selector used
when their styles are generated.
```

Reply:
Either option works for me. Given the current article used matches all the other block API READMEs and we're referencing it as the only block API that allows CSS selector customizations, "the" is arguably more appropriate, so I'll keep it as it is for now.

Reply:
I should note, as far as we are still supporting `__experimentalSelector` properties, I'm considering those as part of the Block Selectors API despite them residing within `supports`.